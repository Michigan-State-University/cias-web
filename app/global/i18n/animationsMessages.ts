import { defineMessages } from 'react-intl';

import { NarratorAnimation } from 'models/Narrator';

export const scope = 'app.global.Animations';

export default defineMessages<NarratorAnimation>({
  uncertain: {
    id: `${scope}.uncertain`,
    defaultMessage: 'Uncertain',
  },
  wave: {
    id: `${scope}.wave`,
    defaultMessage: 'Wave',
  },
  greet: {
    id: `${scope}.greet`,
    defaultMessage: 'Greet',
  },
  otohLeftHand: {
    id: `${scope}.otohLeftHand`,
    defaultMessage: 'On the other hand (left)',
  },
  otohRightHand: {
    id: `${scope}.otohRightHand`,
    defaultMessage: 'On the other hand (right)',
  },
  pointRight: {
    id: `${scope}.pointRight`,
    defaultMessage: 'Point right',
  },
  pointLeft: {
    id: `${scope}.pointLeft`,
    defaultMessage: 'Point left',
  },
  pointUp: {
    id: `${scope}.pointUp`,
    defaultMessage: 'Point up',
  },
  pointDown: {
    id: `${scope}.pointDown`,
    defaultMessage: 'Point down',
  },
  pointLeftUp: {
    id: `${scope}.pointLeftUp`,
    defaultMessage: 'Point left up',
  },
  pointRightUp: {
    id: `${scope}.pointRightUp`,
    defaultMessage: 'Point right up',
  },
  explain: {
    id: `${scope}.explain`,
    defaultMessage: 'Explain',
  },
  rest: {
    id: `${scope}.rest`,
    defaultMessage: 'Rest',
  },
  restShoulderRubbing: {
    id: `${scope}.restShoulderRubbing`,
    defaultMessage: 'Rest shoulder rubbing',
  },
  restStretching: {
    id: `${scope}.restStretching`,
    defaultMessage: 'Rest stretching',
  },
  restWeightShift: {
    id: `${scope}.restWeightShift`,
    defaultMessage: 'Rest weight shift',
  },
  read: {
    id: `${scope}.read`,
    defaultMessage: 'Read',
  },
  reading: {
    id: `${scope}.reading`,
    defaultMessage: 'Reading',
  },
  confused: {
    id: `${scope}.confused`,
    defaultMessage: 'Confused',
  },
  doMagic: {
    id: `${scope}.doMagic`,
    defaultMessage: 'Do magic',
  },
  getAttention: {
    id: `${scope}.getAttention`,
    defaultMessage: 'Get attention',
  },
  search: {
    id: `${scope}.search`,
    defaultMessage: 'Search',
  },
  suggest: {
    id: `${scope}.suggest`,
    defaultMessage: 'Suggest',
  },
  surprised: {
    id: `${scope}.surprised`,
    defaultMessage: 'Surprised',
  },
  congratulate: {
    id: `${scope}.congratulate`,
    defaultMessage: 'Congratulate',
  },
  flyIn: {
    id: `${scope}.flyIn`,
    defaultMessage: 'Fly in',
  },
  flyOut: {
    id: `${scope}.flyOut`,
    defaultMessage: 'Fly out',
  },
  standStill: {
    id: `${scope}.standStill`,
    defaultMessage: 'Stand still',
  },
  process: {
    id: `${scope}.process`,
    defaultMessage: 'Process',
  },
  listen: {
    id: `${scope}.listen`,
    defaultMessage: 'Listen',
  },
  think: {
    id: `${scope}.think`,
    defaultMessage: 'Think',
  },
  write: {
    id: `${scope}.write`,
    defaultMessage: 'Write',
  },
  idle: {
    id: `${scope}.idle`,
    defaultMessage: 'Idle',
  },
  turnPage: {
    id: `${scope}.turnPage`,
    defaultMessage: 'Turn page',
  },
  openBook: {
    id: `${scope}.openBook`,
    defaultMessage: 'Open book',
  },
  closeBook: {
    id: `${scope}.closeBook`,
    defaultMessage: 'Close book',
  },
  announce: {
    id: `${scope}.announce`,
    defaultMessage: 'Announce',
  },
  blink: {
    id: `${scope}.blink`,
    defaultMessage: 'Blink',
  },
  acknowledge: {
    id: `${scope}.acknowledge`,
    defaultMessage: 'Acknowledge',
  },
  acknowledgeThoughtful: {
    id: `${scope}.acknowledgeThoughtful`,
    defaultMessage: 'Acknowledge thoughtful',
  },
  browsUp: {
    id: `${scope}.browsUp`,
    defaultMessage: 'Brows up',
  },
  decline: {
    id: `${scope}.decline`,
    defaultMessage: 'Decline',
  },
  declineAnnoyed: {
    id: `${scope}.declineAnnoyed`,
    defaultMessage: 'Decline annoyed',
  },
  declineThoughtful: {
    id: `${scope}.declineThoughtful`,
    defaultMessage: 'Decline thoughtful',
  },
  pleased: {
    id: `${scope}.pleased`,
    defaultMessage: 'Pleased',
  },
  sad: {
    id: `${scope}.sad`,
    defaultMessage: 'Sad',
  },
  hearLeftEar: {
    id: `${scope}.hearLeftEar`,
    defaultMessage: 'Hear left ear',
  },
  hearRightEar: {
    id: `${scope}.hearRightEar`,
    defaultMessage: 'Hear right ear',
  },
  hearBothEars: {
    id: `${scope}.hearBothEars`,
    defaultMessage: 'Hear both ears',
  },
  yawn: {
    id: `${scope}.yawn`,
    defaultMessage: 'Yawn',
  },
  eatCracker: {
    id: `${scope}.eatCracker`,
    defaultMessage: 'Eat cracker',
  },
  wearSunglasses: {
    id: `${scope}.wearSunglasses`,
    defaultMessage: 'Wear sunglasses',
  },
  lookDown: {
    id: `${scope}.lookDown`,
    defaultMessage: 'Look down',
  },
  lookDownAndBlink: {
    id: `${scope}.lookDownAndBlink`,
    defaultMessage: 'Look down and blink',
  },
  glanceUp: {
    id: `${scope}.glanceUp`,
    defaultMessage: 'Glance up',
  },
  glanceDown: {
    id: `${scope}.glanceDown`,
    defaultMessage: 'Glance down',
  },
  glanceLeft: {
    id: `${scope}.glanceLeft`,
    defaultMessage: 'Glance left',
  },
  glanceRight: {
    id: `${scope}.glanceRight`,
    defaultMessage: 'Glance right',
  },
  moveLeft: {
    id: `${scope}.moveLeft`,
    defaultMessage: 'Move left',
  },
  moveRight: {
    id: `${scope}.moveRight`,
    defaultMessage: 'Move right',
  },
});
