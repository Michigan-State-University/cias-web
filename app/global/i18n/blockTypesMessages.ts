import { defineMessages } from 'react-intl';

export const scope = 'app.global.BlockTypes';

export default defineMessages({
  BodyAnimation: {
    id: `${scope}.BodyAnimation`,
    defaultMessage: 'Body Animation',
  },
  Speech: {
    id: `${scope}.Speech`,
    defaultMessage: 'Speech',
  },
  Reflection: {
    id: `${scope}.Reflection`,
    defaultMessage: 'Speech',
  },
  ReflectionFormula: {
    id: `${scope}.ReflectionFormula`,
    defaultMessage: 'Speech',
  },
  HeadAnimation: {
    id: `${scope}.HeadAnimation`,
    defaultMessage: 'Head Animation',
  },
  ReadQuestion: {
    id: `${scope}.ReadQuestion`,
    defaultMessage: 'Read question',
  },
  Pause: {
    id: `${scope}.Pause`,
    defaultMessage: 'Pause',
  },
  Feedback: {
    id: `${scope}.Feedback`,
    defaultMessage: 'Show Spectrum',
  },
});
