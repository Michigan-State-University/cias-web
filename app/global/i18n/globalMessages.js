import { defineMessages } from 'react-intl';

export const scope = 'app.GlobalMessages';

export default defineMessages({
  variables: {
    variableNamePlaceholder: {
      id: `${scope}.variableNamePlaceholder`,
      defaultMessage: 'Variable name...',
    },
    variableScorePlaceholder: {
      id: `${scope}.variableScorePlaceholder`,
      defaultMessage: 'Score',
    },
    value: {
      id: `${scope}.value`,
      defaultMessage: 'Value:',
    },
    emptyBadge: {
      id: `${scope}.emptyBadge`,
      defaultMessage: 'No assigned value',
    },
  },
  blockTypes: {
    BodyAnimation: {
      id: `${scope}.BodyAnimation`,
      defaultMessage: 'Body Animation',
    },
    Speech: {
      id: `${scope}.Speech`,
      defaultMessage: 'Speech',
    },
    Reflection: {
      id: `${scope}.Reflection`,
      defaultMessage: 'Speech',
    },
    HeadAnimation: {
      id: `${scope}.HeadAnimation`,
      defaultMessage: 'Head Animation',
    },
    ReadQuestion: {
      id: `${scope}.ReadQuestion`,
      defaultMessage: 'Read question',
    },
  },
  questionTypes: {
    'Question::Single': {
      id: `${scope}.Question::Single`,
      defaultMessage: 'Single answer screen',
    },
    'Question::Multiple': {
      id: `${scope}.Question::Multiple`,
      defaultMessage: 'Multi answer screen',
    },
    'Question::TextBox': {
      id: `${scope}.Question::TextBox`,
      defaultMessage: 'Textbox screen',
    },
    'Question::Number': {
      id: `${scope}.Question::Number`,
      defaultMessage: 'Number screen',
    },
    'Question::Grid': {
      id: `${scope}.Question::Grid`,
      defaultMessage: 'Grid screen',
    },
    'Question::AnalogueScale': {
      id: `${scope}.Question::AnalogueScale`,
      defaultMessage: 'Visual Analogue Scale screen',
    },
    'Question::Information': {
      id: `${scope}.Question::Information`,
      defaultMessage: 'Information screen',
    },
    'Question::Url': {
      id: `${scope}.Question::Url`,
      defaultMessage: 'Url screen',
    },
    'Question::FollowUpContact': {
      id: `${scope}.Question::FollowUpContact`,
      defaultMessage: 'FollowUpContact screen',
    },
  },
  statuses: {
    draft: {
      id: `${scope}.draft`,
      defaultMessage: 'Draft',
    },
    published: {
      id: `${scope}.published`,
      defaultMessage: 'Published',
    },
    closed: {
      id: `${scope}.closed`,
      defaultMessage: 'Closed',
    },
    archived: {
      id: `${scope}.archived`,
      defaultMessage: 'Archived',
    },
  },
  createProblemError: {
    id: `${scope}.createProblemError`,
    defaultMessage: `Couldn't create an intervention`,
  },
  editProblemError: {
    id: `${scope}.editProblemError`,
    defaultMessage: `Couldn't edit an intervention`,
  },
});
