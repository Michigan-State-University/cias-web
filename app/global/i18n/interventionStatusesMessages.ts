import { defineMessages } from 'react-intl';

export const scope = 'app.global.InterventionStatusesMessages';

export default defineMessages({
  draft: {
    id: `${scope}.draft`,
    defaultMessage: 'Draft',
  },
  published: {
    id: `${scope}.published`,
    defaultMessage: 'Published',
  },
  closed: {
    id: `${scope}.closed`,
    defaultMessage: 'Closed',
  },
  archived: {
    id: `${scope}.archived`,
    defaultMessage: 'Archived',
  },
});
