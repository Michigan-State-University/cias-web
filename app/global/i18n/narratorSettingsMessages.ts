import { defineMessages } from 'react-intl';

import { NarratorSettingsKey } from 'models/Narrator';

export const scope = 'app.global.NarratorSettings';

export default defineMessages<NarratorSettingsKey>({
  [NarratorSettingsKey.VOICE]: {
    id: `${scope}.voice`,
    defaultMessage: 'Voice',
  },
  [NarratorSettingsKey.ANIMATION]: {
    id: `${scope}.animation`,
    defaultMessage: 'Display Narrator',
  },
  [NarratorSettingsKey.EXTRA_SPACE_FOR_NARRATOR]: {
    id: `${scope}.extra_space_for_narrator`,
    defaultMessage: 'Extra Space for Narrator',
  },
  [NarratorSettingsKey.CHARACTER]: {
    id: `${scope}.character`,
    defaultMessage: 'Character',
  },
});
