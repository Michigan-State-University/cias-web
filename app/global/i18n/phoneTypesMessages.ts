import { defineMessages } from 'react-intl';

import { PhoneType } from 'models/HfhsPatient';

export const scope = 'app.global.PhoneTypes';

export default defineMessages<PhoneType>({
  home: {
    id: `${scope}.home`,
    defaultMessage: 'Home',
  },
  work: {
    id: `${scope}.work`,
    defaultMessage: 'Work',
  },
  mobile: {
    id: `${scope}.mobile`,
    defaultMessage: 'Mobile',
  },
});
