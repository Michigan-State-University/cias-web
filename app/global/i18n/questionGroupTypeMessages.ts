import { defineMessages } from 'react-intl';

import { GroupType } from 'models/QuestionGroup';

export const scope = 'app.global.QuestionGroupType';

export default defineMessages<GroupType>({
  [GroupType.PLAIN]: {
    id: `${scope}.QuestionGroup::Plain`,
    defaultMessage: 'Plain',
  },
  [GroupType.FINISH]: {
    id: `${scope}.QuestionGroup::Finish`,
    defaultMessage: 'Finish',
  },
  [GroupType.TLFB]: {
    id: `${scope}.QuestionGroup::Tlfb`,
    defaultMessage: 'Timeline Followback',
  },
});
