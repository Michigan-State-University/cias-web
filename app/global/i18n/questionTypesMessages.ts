import { defineMessages } from 'react-intl';

import { QuestionTypes } from 'models/Question';

export const scope = 'app.global.QuestionTypes';

export default defineMessages<QuestionTypes>({
  [QuestionTypes.SINGLE]: {
    id: `${scope}.Question::Single`,
    defaultMessage: 'Single answer',
  },
  [QuestionTypes.MULTIPLE]: {
    id: `${scope}.Question::Multiple`,
    defaultMessage: 'Multi answer',
  },
  [QuestionTypes.FREE_RESPONSE]: {
    id: `${scope}.Question::FreeResponse`,
    defaultMessage: 'Free Response',
  },
  [QuestionTypes.PHONE]: {
    id: `${scope}.Question::Phone`,
    defaultMessage: 'Phone Text Messaging',
  },
  [QuestionTypes.NAME]: {
    id: `${scope}.Question::Name`,
    defaultMessage: 'Name',
  },
  [QuestionTypes.PARTICIPANT_REPORT]: {
    id: `${scope}.Question::ParticipantReport`,
    defaultMessage: 'Participant Report',
  },
  [QuestionTypes.NUMBER]: {
    id: `${scope}.Question::Number`,
    defaultMessage: 'Number',
  },
  [QuestionTypes.GRID]: {
    id: `${scope}.Question::Grid`,
    defaultMessage: 'Grid',
  },
  [QuestionTypes.SLIDER]: {
    id: `${scope}.Question::Slider`,
    defaultMessage: 'Slider',
  },
  [QuestionTypes.INFORMATION]: {
    id: `${scope}.Question::Information`,
    defaultMessage: 'Information Only',
  },
  [QuestionTypes.EXTERNAL_LINK]: {
    id: `${scope}.Question::ExternalLink`,
    defaultMessage: 'External Link',
  },
  [QuestionTypes.FEEDBACK]: {
    id: `${scope}.Question::Feedback`,
    defaultMessage: 'Feedback',
  },
  [QuestionTypes.FINISH]: {
    id: `${scope}.Question::Finish`,
    defaultMessage: 'Finish',
  },
  [QuestionTypes.DATE]: {
    id: `${scope}.Question::Date`,
    defaultMessage: 'Date',
  },
  [QuestionTypes.CURRENCY]: {
    id: `${scope}.Question::Currency`,
    defaultMessage: 'Currency',
  },
  [QuestionTypes.THIRD_PARTY]: {
    id: `${scope}.Question::ThirdParty`,
    defaultMessage: 'Third Party Report',
  },
  [QuestionTypes.TLFB_CONFIG]: {
    id: `${scope}.Question::TlfbConfig`,
    defaultMessage: 'TLFB Config',
  },
  [QuestionTypes.TLFB_EVENTS]: {
    id: `${scope}.Question::TlfbEvents`,
    defaultMessage: 'TLFB Events',
  },
  [QuestionTypes.TLFB_QUESTION]: {
    id: `${scope}.Question::TlfbQuestion`,
    defaultMessage: 'TLFB Questions',
  },
  [QuestionTypes.HENRY_FORD_QUESTION]: {
    id: `${scope}.Question::HenryFord`,
    defaultMessage: 'Henry Ford Question',
  },
  [QuestionTypes.HENRY_FORD_INITIAL]: {
    id: `${scope}.Question::HenryFordInitial`,
    defaultMessage: 'Henry Ford Initial Screen',
  },
});
