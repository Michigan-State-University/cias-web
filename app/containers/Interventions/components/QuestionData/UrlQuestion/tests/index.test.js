import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { IntlProvider } from 'react-intl';
import { DEFAULT_LOCALE } from 'i18n';
import { QuestionUrlWithIntl as UrlQuestion } from '../index';

const defaultProps = {
  updateUrl: jest.fn(),
  selectedQuestion: {
    body: {
      variable: { name: '' },
      data: [{ payload: 'www.google.com' }],
    },
  },
};

describe('<UrlQuestion />', () => {
  it('should match the snapshot', () => {
    const renderedComponent = renderer
      .create(
        <IntlProvider locale={DEFAULT_LOCALE}>
          <UrlQuestion {...defaultProps} />
        </IntlProvider>,
      )
      .toJSON();
    expect(renderedComponent).toMatchSnapshot();
  });
});
