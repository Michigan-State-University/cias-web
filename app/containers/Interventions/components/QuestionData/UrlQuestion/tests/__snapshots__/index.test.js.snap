// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UrlQuestion /> should match the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
  margin-top: 10px;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  margin-bottom: 10px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  width: 100%;
}

.c3 {
  padding: 12px;
  border: 1px solid #E9F1FB;
  border-radius: 5px;
  width: 153px;
  max-width: 200px;
  padding-top: 12px;
  padding-bottom: 12px;
  color: rgba(16,121,105,1);
  background-color: transparent;
  border: 1px solid transparent;
  text-align: center;
}

.c3:focus {
  box-shadow: none;
  outline: none;
  border: 1px solid #107969;
}

.c8 {
  resize: none;
  padding: 12px;
  border: 1px solid #E9F1FB;
  border-radius: 5px;
  margin-right: 9px;
  width: 100%;
  height: auto;
  background-color: transparent;
  border: 1px solid transparent;
}

.c8:focus {
  box-shadow: none;
  outline: none;
  border: 1px solid #107969;
}

.c4 {
  width: auto;
  height: auto;
  display: block;
  border-radius: 5px;
  width: 100%;
  padding-top: 14px;
  padding-bottom: 14px;
  padding-right: 21px;
  padding-left: 21px;
  background-color: rgba(233,241,251,1);
}

.c9 {
  width: auto;
  height: auto;
  display: block;
  border-radius: 5px;
  margin-top: 5px;
  margin-bottom: 5px;
}

.c11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: auto;
  width: 50px;
  height: 50px;
}

.c11:after {
  content: ' ';
  display: block;
  width: 33.33333333333333px;
  height: 33.33333333333333px;
  border-radius: 50%;
  border: 6px solid #C866EA;
  border-color: #C866EA transparent #C866EA transparent;
  -webkit-animation: ixzqdH 1.2s linear infinite;
  animation: ixzqdH 1.2s linear infinite;
}

.c10 {
  width: 100%;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c2 {
  padding: 5px 10px;
  font-size: 13px;
  font-weight: 600;
  border-radius: 4px;
  background-color: rgba(16,121,105,0.1);
  color: rgba(16,121,105,1);
}

.c2:focus {
  box-shadow: none;
  outline: none;
  border: 1px solid transparent;
}

<div
  className="c0"
>
  <div
    className="c1"
    display="flex"
  >
    <input
      className="c2 c3"
      color="#107969"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      opacity={0.1}
      placeholder="Variable name..."
      type="singleline"
      value=""
      width="153px"
    />
  </div>
  <div
    className="c4"
    width="100%"
  >
    <div
      className="c5"
    >
      <div
        className="c6"
        width="100%"
      >
        <div
          className="c7"
        >
          <textarea
            className="c8"
            height="auto"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Paste here a web page link to show it to a respondend."
            rows="3"
            value="www.google.com"
            width="100%"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="c9"
  >
    <div
      className="c9"
    >
      <div
        className="c5"
      >
        <div
          className="c10"
          data-testid="inline-loader"
        >
          <div
            className="c11"
            color="#C866EA"
            size={50}
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
